foladipo/docs-cabinet-cp2
# Feature/148100401/enable searching for documents

# NOTE
This branch was misnamed when created. The right name should have been `feature/148098733/enable-searching-for-users`. Unfortunately, this mistake was not noticed until it after it had been merged with `develop`. The branch has been renamed though.



#### What does this PR do?

This PR enables a user to search for other users.

#### Description of task to be completed

Enable a user to search for other users.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have sign up for a new account.

- Open the side menu and click the `"Search for users"` option. You will then be taken to search page.

- Search away! Note that you can search based on part or all of a user's first name, last name or username.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148098733

#### Screenshots
<img width="1280" alt="screen shot 2017-07-08 at 11 32 30 am" src="https://user-images.githubusercontent.com/26963575/27989123-9b216b8e-6429-11e7-9245-fa7c3841128d.png">
